server/api/submission/submission.integration.js

```diff
@@ -2,10 +2,68 @@
 
 var app = require('../..');
 var request = require('supertest');
-
+var User = require('../user/user.model')
```

Comment:
In non-es6 javascript semicolons should be enforced because they can change the behavior of simple expressions